.github/workflows/linux.yml

```diff
@@ -75,7 +74,8 @@ jobs:
 
     - name: Install packages
       run: |
-        python3.9 -m pip install --user numpy setuptools wheel cython pytest pytest-xdist pybind11 pytest-xdist pythran
+        python3.9 -m pip install --user -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
+        python3.9 -m pip install --user setuptools wheel cython pytest pytest-xdist pybind11 pytest-xdist pythran
```

Comment:
`pytest-xdist` is included twice here, can you remove one?

Reply:
Yep, done in 85a5d355271dc3597dd0f16a62a8f650cc896187